starter/cloudstorage/src/main/java/com/udacity/jwdnd/course1/cloudstorage/data/Message.java

```diff
@@ -0,0 +1,36 @@
+package com.udacity.jwdnd.course1.cloudstorage.data;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class Message {
+
+    private boolean error;
+    private boolean success;
+    private String msg;
+
```

Comment:
Add three arg constructor.
```suggestion
public Message(boolean error, boolean success, String msg) {
        this.error = error;
        this.success = success;
        this.msg = msg;
    }
```